be/test/storage/rowset_column_partial_update_test.cpp

```diff
@@ -185,8 +186,20 @@ class RowsetColumnPartialUpdateTest : public ::testing::Test, testing::WithParam
                 idx++;
             }
         }
-        for (int i = 0; i < segment_num; i++) {
-            CHECK_OK(writer->flush_chunk(*chunk));
+        if (spilt_keys) {
+            for (int i = 0; i < segment_num; i++) {
+                auto tmp_chunk = ChunkHelper::new_chunk(schema, keys.size() / segment_num);
+                std::vector<uint32_t> indexes;
+                for (int j = i; j < chunk->num_rows(); j += segment_num) {
+                    indexes.push_back(j);
+                }
+                tmp_chunk->append_selective(*chunk, indexes.data(), 0, indexes.size());
+                CHECK_OK(writer->flush_chunk(*tmp_chunk));
```

Comment:
In the RELEASE mode, CHECK_OK doesn't work.

Reply:
Ok, but this is UT code only.